src/interp.rs

```diff
@@ -29,21 +29,8 @@ trait Interpolator: std::fmt::Debug {
 pub struct Interp(Box<dyn Interpolator>);
 
 impl Interp {
-    pub fn new(taps: usize, factor: usize, channels: u32) -> Self {
-        let imp: Box<dyn Interpolator> = match (taps, factor, channels) {
-            (49, 2, 1) => Box::new(specialized::Interp2F::<[f32; 1]>::new()),
-            (49, 2, 2) => Box::new(specialized::Interp2F::<[f32; 2]>::new()),
-            (49, 2, 4) => Box::new(specialized::Interp2F::<[f32; 4]>::new()),
-            (49, 2, 6) => Box::new(specialized::Interp2F::<[f32; 6]>::new()),
-            (49, 2, 8) => Box::new(specialized::Interp2F::<[f32; 8]>::new()),
-            (49, 4, 1) => Box::new(specialized::Interp4F::<[f32; 1]>::new()),
-            (49, 4, 2) => Box::new(specialized::Interp4F::<[f32; 2]>::new()),
-            (49, 4, 4) => Box::new(specialized::Interp4F::<[f32; 4]>::new()),
-            (49, 4, 6) => Box::new(specialized::Interp4F::<[f32; 6]>::new()),
-            (49, 4, 8) => Box::new(specialized::Interp4F::<[f32; 8]>::new()),
-            (taps, factor, channels) => Box::new(generic::Interp::new(taps, factor, channels)),
-        };
-        Self(imp)
+    pub fn new(_taps: usize, _factor: usize, _channels: u32) -> Self {
+        unimplemented!()
```

Comment:
This suggests that this commit should be squashed with another one :) This alone doesn't seem runnable as-is.